.github/workflows/github-pr.yml

```
@@ -18,10 +18,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.4.x]
+        node-version: [22]
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v34
```

Is this supposed to be v4?

```suggestion
        uses: actions/checkout@v4
```

---

> Is this supposed to be v4? It appears to be failing CI [here](https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/actions/runs/11712676678/job/32635278261?pr=148)

Yep, sorry about that.  And thank you for creating this extension :)